pkg/cfn/builder/managed_nodegroup.go

```diff
@@ -102,6 +102,11 @@ func (m *ManagedNodeGroupResourceSet) AddAllResources() error {
 		return gfnt.NewString(getAMIType(selectManagedInstanceType(m.nodeGroup)))
 	}
 
+	var efaSG *gfnt.Value
+	if api.IsEnabled(m.nodeGroup.EFAEnabled) {
+		desc := "worker nodes in group " + m.nodeGroup.Name
+		efaSG = m.addEFASecurityGroup(m.vpcImporter.VPC(), m.clusterConfig.Metadata.Name, desc)
+	}
```

Comment:
can I ask why we are doing a `api.IsEnabled` here and passing the value into the `makeLaunchTemplateData` method, when the method already has some `api.IsEnabled` going on?

Reply:
good spot, moved it.